tasks/fun-chat/README.md

```diff
@@ -0,0 +1,154 @@
+## Task «Fun Chat»
```

Comment:
Please move the description into this folder: https://github.com/rolling-scopes-school/tasks/tree/master/stage2/tasks. @dzmitry-varabei and I discussed that it would be better to distribute new tasks by the stage at which they are assigned.

Reply:
done